src/wallet/test/fuzz/notifications.cpp

```diff
@@ -117,9 +117,6 @@ FUZZ_TARGET(wallet_notifications, .init = initialize_setup)
                     tx.vout.emplace_back(in, wallet.GetScriptPubKey(fuzzed_data_provider));
                     // Add tx to block
                     block.vtx.emplace_back(MakeTransactionRef(tx));
-                    // Check that funding the tx doesn't crash the wallet
-                    a.FundTx(fuzzed_data_provider, tx);
```

Comment:
`FundTx` is part of `FuzzedWallet` and is only used in this target. Since we're removing this call here, maybe we could delete the entire function in FuzzedWallet.

Reply:
> IIUC the overhead comes from the knapsack solver running 1000 times over all inputs, which seems heavy. I wonder if there is an easy way to disable knapsack here?

This is the 1000 iterations from `ApproximateBestSubset`?

Reply:
Just fyi - When I was working on this paper (https://repositorio.usp.br/directbitstream/ceb3347f-d97b-4995-8c0d-17ec195dd128/3218430.pdf), I did some experiments by reducing the number of iterations in `ApproximateBestSubset`, I think I mention it in the paper, but I remember that reducing the number of iterations by the half had no impact on its behavior.

Reply:
Keeping the two `FundTx` calls doesn't make the target significantly slower for me (`-runs=1` takes 8 sec vs 6 sec without them). But I agree that if they are removed then the function should be removed from `FuzzedWallet`.

Reply:
Removing Knapsack is still on my to-do list!